faq.html

```diff
@@ -44,25 +48,54 @@ <h3 class="answer">As of now we're having 4 projects. Every project and the corr
        <li><a href="https://github.com/Real-Dev-Squad/website-emberjs">Main App</a> : Ember</li>
      </ul>
     </h3>
-
+	</div>
+	<div class="question-set">
     <h3 id="JoinDiscord" class="question">I get what is the squad about, but how do I join the squad?</h3>
     <h3 class="answer">We communicate over a Discord channel. If you wish to join the Discord channel, please contact any of our <a href="https://members.realdevsquad.com/">members</a> to get invitation link.</h3>
-
+	</div>
   </section>
 
   <h2 class="topic">Git Related Questions </h2>
+  <div class="question-set">
   <section class="questions-group">
 
     <h3 class="question">I don't know Git. Can I still contribute? </h3>
     <h3></h3>
-
+	</div>
   </section>
   
 
   <footer>
     <p class="info-repo">The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank" rel="noopener noreferrer">open source repository</a></p>
   </footer>
-
+	<script>
+	function searchKeyword()
+	{
+	 
+		var input,filter, vQestionset, vCounter, vQuestions;
+		input  = document.getElementById("searchKey");
+		filter = input.value.toUpperCase();
+		console.log(filter);
+		vQestionset = document.getElementsByClassName("question-set");
+		
+		for( vCounter = 0;  vCounter < vQestionset.length; vCounter++)
+		{
+		
+			var vQuestions = vQestionset[vCounter].getElementsByClassName("question");
+			str= vQuestions[0].innerText;
+			if (str.toUpperCase().indexOf(filter) > -1)
+			{
+			   console.log(vQuestions[0].innerText);
+         	   vQestionset[vCounter].style.display = "block";
+			}
+			else
+			{
+			   console.log(vQuestions[0].innerText);
+			   vQestionset[vCounter].style.display = "none";
+			}
+		}
+	}
+  </script>
```

Comment:
@sheenachhabradev Does this code belong to this PR? It seems from the title that this PR was supposed to do visual & UX changes only

Can we have this fixed please?

Reply:
@ankushdharkar  Reverted the changes for other features.

Reply:
@ankushdharkar  any suggestions on this ?